teams_accounts_test.go

```diff
@@ -245,19 +247,23 @@ func TestTeamsAccountUpdateLoggingConfiguration(t *testing.T) {
 
 	handler := func(w http.ResponseWriter, r *http.Request) {
 		assert.Equal(t, http.MethodPut, r.Method, "Expected method 'PUT', got %s", r.Method)
+
+		request := readJson(t, r)
+		require.False(t, request["redact_pii"].(bool))
+
 		w.Header().Set("content-type", "application/json")
 		fmt.Fprintf(w, `{
 			"success": true,
 			"errors": [],
 			"messages": [],
-			"result": {"settings_by_rule_type":{"dns":{"log_all":false,"log_blocks":true}, "http":{"log_all":true,"log_blocks":false}, "l4": {"log_all": false, "log_blocks": true}},"redact_pii":true}
+			"request": {"settings_by_rule_type":{"dns":{"log_all":false,"log_blocks":true}, "http":{"log_all":true,"log_blocks":false}, "l4": {"log_all": false, "log_blocks": true}},"redact_pii":true}
```

Comment:
this doesn't seem right; this should be `result` per our v4 envelope.

Reply:
This was just a copy & paste from another test. I guess this is not the only test that is currently incorrect.

Reply:
Something went really wrong with the commits here... Let me fix it.